.travis.yml

```diff
@@ -2,18 +2,39 @@ sudo: false
 language: php
 php:
   - 7.2
-
-branches:
-  only: 
-    - master
+services:
+  - docker
+addons:
+  apt:
+    packages:
+      - docker-ce
+env:
+  - DOCKER_COMPOSE_VERSION=1.23.2
```

Comment:
I'd just hardcode this, similarly to the Node version.

Reply:
done!